### PR TITLE
refactor(Prambors Radio): drop streaming.pramborsfm.com support, removed song info in the presence, replaced 'live' image key with 'play'

### PR DESCRIPTION
**Pull request by Allvaa**
-  **Drop streaming.pramborsfm.com support**
The page was gone
- **Removed song info in the presence**
It's often out of sync with what's playing, and I find the uppercase title is annoying. removing it, now I think it's much cleaner
image
- **Replaced 'live' image key with 'play'**

### CODE REVIEW

File: websites/P/Prambors Radio/presence.ts [hunk 1]
```diff
@@ -2,75 +2,43 @@ const presence = new Presence({
...
 
 presence.on("UpdateData", async () => {
-  const pause = (await strings).pause;
-  if (
-    ["streaming.pramborsfm.com", "live.pramborsfm.com"].includes(
-      document.location.hostname
-    )
-  ) {
+  const { play, pause } = await strings;
+  if (document.location.hostname === "live.pramborsfm.com") {
```
Comment (EncryptedDev) [inline websites/P/Prambors Radio/presence.ts:16]: Whats the point of this check when you do not support `pramborsfm.com`?

Comment (Allvaa) [reply websites/P/Prambors Radio/presence.ts:16]: yes it doesnt support pramborsfm.com, but it supports www.pramborsfm.com.
I didn't do anything to the main page so it's just like this
image

Comment (Slowlife01) [reply websites/P/Prambors Radio/presence.ts:16]: why bother add that domain then?

Comment (Allvaa) [reply websites/P/Prambors Radio/presence.ts:16]: to show that you open www.pramborsfm.com 🤔

Comment (Allvaa) [reply websites/P/Prambors Radio/presence.ts:16]: hi, so i need to remove www.pramborsfm.com?

Comment (EncryptedDev) [reply websites/P/Prambors Radio/presence.ts:16]: If you have no support to it, yes

Comment (Allvaa) [reply websites/P/Prambors Radio/presence.ts]: weird that it was merged on my previous pr

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

**Comment by EncryptedDev** [comment]
Fix conflicts

**Pull request closed without merging**